sweep/fee_bumper.go

```diff
@@ -124,6 +124,10 @@ type BumpRequest struct {
 	// StartingFeeRate is an optional parameter that can be used to specify
 	// the initial fee rate to use for the fee function.
```

Comment:
may help with review just to say in the commit msg why it will not work with the upcoming change? just cause this commit still does allow immediate broadcast if `Immediate` is true so it is unclear just looking at this commit why that then would work later?

Reply:
cool updated!